README.adoc

```diff
@@ -143,40 +157,71 @@ method. In a real development environment, you would want to orchestrate much mo
+// Adding health checks to the inventory service
 // =================================================================================================
 
-=== Adding a health check to the inventory service
+=== Adding health checks to the inventory service
 
 [role="code_command hotspot file=0", subs="quotes"] 
 ---- 
-#Create the `InventoryHealth` class.#
-`src/main/java/io/openliberty/guides/inventory/InventoryHealth.java`
+#Create the `InventoryReadinessCheck` class.#
+`src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java`
 ---- 
 
-InventoryHealth.java
+InventoryReadinessCheck.java
 [source, java, linenums, role='code_column']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/InventoryHealth.java[tags=InventoryHealth]
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[tags=InventoryReadinessCheck]
 ----
 
 This time, you are checking whether or not the service is in maintenance or if it's down. 
-For simplicity, the custom [hotspot=3 file=1]`io_openliberty_guides_inventory_inMaintenance` 
-MicroProfile Config property defined in the [hotspot file=1]`resources/CustomConfigSource.json` 
+For simplicity, the custom [hotspot=3 file=2]`io_openliberty_guides_inventory_inMaintenance` 
+MicroProfile Config property defined in the [hotspot file=2]`resources/CustomConfigSource.json` 
 file is used to indicate whether the service is in maintenance or not. This file has already 
 been created for you. To check if the service is down, simply make a HTTP GET request to 
 the `system` service and check the status returned by the response. You make a GET request 
 to the `system` service rather than the `inventory` service because the `inventory` service 
 depends on the `system` service. In other words, the `inventory` service wouldn't work if 
 the `system` service is down. If the status is not 200, then the service is not running. 
-Based on these two factors, the [hotspot=19-37 file=0]`isHealthy()` method returns whether 
+Based on these two factors, the [hotspot=19-36 file=0]`isHealthy()` method returns whether 
 or not the `inventory` service is healthy.
 
 If you are curious about the injected [hotspot=17 file=0]`inventoryConfig` object or if 
 you want more information on MicroProfile Config, see 
 https://openliberty.io/guides/microprofile-config.html[Configuring microservices^].
 
+[role="code_command hotspot file=1", subs="quotes"] 
+---- 
+#Create the `InventoryLivenessCheck` class.#
+`src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java`
+---- 
+
+InventoryLivenessCheck.java
+[source, java, linenums, role='code_column']
+----
+include::finish/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[tags=InventoryLivenessCheck]
+----
+
+Just like the `system` liveness check, you are checking the heap memory usage. If more 
+than 90% of the maximum memory is being used, a status of `DOWN` will be returned.  
```

Comment:
Change
"Just like the `system` liveness check, you are checking the heap memory usage. If more than 90% of the maximum memory is being used, a status of `DOWN` will be returned."
To
"As with the `system` liveness check, you are checking the heap memory usage. If more than 90% of the maximum memory is being used, a `DOWN` status is returned. "